src/workflow/ParameterManager.py

```diff
@@ -0,0 +1,115 @@
+import pyopenms as poms
+import json
+import shutil
+import streamlit as st
```

Comment:
same here... try to make class less dependent on the global st state but only set it (if possible) during construction. This makes it much easier to reason about the lifetime of these objects and references.